resolwe_bio/processes/workflows/bbduk_star.py

```diff
@@ -212,10 +212,91 @@ class ChimericReadsOptions:
                     "and used --chimSegmentMin 20, a chimeric alignment with 130b on one chromosome "
                     "and 20b on the other will be output, while 135 + 15 won't be.",
                 )
+
+                chim_out_type = StringField(
+                    label="Chimeric output type [--chimOutType]",
+                    default="SeparateSAMold",
+                    choices=[
+                        ("WithinBam", "WithinBam"),
+                        ("SeparateSAMold", "SeparateSAMold"),
+                    ],
+                    description="Type of chimeric output produced by STAR.",
+                    disabled="!chimeric_reads.chimeric",
```

Comment:
Based on the hierarchy of input fields, I think the correct reference should be `disabled="!alignment.chimeric_reads.chimeric` since `ChimericReadsOptions` is nested within `Alignment`.

Reply:
Nice catch! I also managed to miss one `disabled="!detect_chimeric.chimeric` therein.